### DISCUSSION TITLE
Mass-selection of repos to add to lists

### DISCUSSION THREAD

**Discussion by MattIPv4** [community/community#8240] [Other Feature Feedback, Questions, & Ideas]
Much like you can mass-select issues or PRs on a repo to perform actions against them, it'd be great if [the stars view](https://github.com/stars/MattIPv4) would allow a user to quickly select multiple repos to add to a list so that folks with many repos can quickly get organized using lists.

**Comment by bailey** [comment]
Totally agree! We'll keep this on the team's radar for when we have bandwidth in the future. Thanks for dropping feedback ✨

**Reply by praptosoehartono** [reply]
@Jokotingkir01
